pupil_src/shared_modules/video_capture/uvc_backend.py

```diff
@@ -180,7 +182,21 @@ def verify_drivers(self):
     def configure_capture(self, frame_size, frame_rate, uvc_controls):
         # Set camera defaults. Override with previous settings afterwards
         if "Pupil Cam" in self.uvc_capture.name:
-            self.ts_offset = 0.0
+            if platform.system() == "Windows":
+                # NOTE: Hardware timestamps seem to be broken on windows. Needs further
+                # investigation! Disabling for now.
+                # TODO: Find accurate offsets for different resolutions!
+                offsets = {"ID0": -0.015, "ID1": -0.015, "ID2": -0.07}
+                match = re.match(r"Pupil Cam\d (?P<cam_id>ID[0-2])", self.name)
+                if not match:
+                    logger.debug(f"Could not parse camera name: {self.name}")
+                    self.ts_offset = 0.0
```

Comment:
Maybe implement a default offset of  `-0.01`? Also apply below for unknown cams
```suggestion
                    self.ts_offset = -0.01
```

Reply:
@papr Hm, shouldn't we maybe use software timestamps for unknown cameras? (This is what this was supposed to be)

Reply:
In this case I'd do:
```suggestion
                    self.ts_offset = None
```

Reply:
`self.ts_offset = None` triggers hardware timestamps. In this specific case, we are on Windows. We do not want to use hardware timestamps in any case on Windows.